docs/swagger.json

```diff
@@ -522,6 +522,39 @@
                 ]
             }
         },
+        "/mentorship_relations/count_pending": {
```

Comment:
I don't think we should modify swagger manually but will confirm that with other mentors

Reply:
Ok, please let me know after confirmation. :)

Reply:
hey the mentors didn't come back yet but I don't want to block you with the task, so let's just leave it here for now :)

Reply:
just received confirmation - we edit the code and the documentation is autogenerated.

Reply:
just received confirmation - we edit the code and the documentation is autogenerated.